code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -1,11 +1,25 @@
 /*
 April 3rd, 2014 marks the day this machine changed the face of the kitchen on NTStation13
 God bless America.
-insert ascii eagle on american flag background here
+          ___----------___
+        _--                ----__
+       -                         ---_
+      -___    ____---_              --_
+  __---_ .-_--   _ O _-                -
+ -      -_-       ---                   -
+-   __---------___                       -
+- _----                                  -
+ -     -_                                 _
+ `      _-                                 _
+       _                           _-_  _-_ _
+      _-                   ____    -_  -   --
+      -   _-__   _    __---    -------       -
+     _- _-   -_-- -_--                        _
+     -_-                                       _
+    _-                                          _
+    -
```

Comment:
Maintainers gonna shouuut at youuuu

Reply:
It's ascii art! The header asked for it!

Reply:
![image](https://user-images.githubusercontent.com/6209658/32473450-92a3b15e-c335-11e7-873f-276f1772a4c9.png)